src/store/index.ts

```diff
@@ -62,18 +64,37 @@ export const addFootPrint = async (spatialId: number) => {
   footPrints.value.push(res.data);
 };
 
-export const removeFootPrint = (spatialId: number) => {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const removeFootPrint = (_spatialId: number) => {
```

Comment:
@annehaley, why this change?

Reply:
Simply to get rid of the style warnings. I set up each of these functions to be filled in, but since they don't use `spatialId` yet, the linter made these suggestions. This can be undone when the variable is used.